Webapp/sources/blueprints/polymer_novel_compound/routes.py

```diff
@@ -0,0 +1,44 @@
+from flask import Response, jsonify, request
+from flask_login import login_required
+from sources import services
+from sources.auxiliary import abort_if_user_not_in_workbook
+
+from . import polymer_novel_compound_bp
+
+
+@polymer_novel_compound_bp.route("/_polymer_novel_compound", methods=["GET", "POST"])
```

Comment:
I wonder if this function could be added as an endpoint to the existing novel_compound blueprints. Any thoughts?